stackinator/etc/envvars.py

```diff
@@ -577,7 +583,7 @@ def meta_impl(args):
         "--prefix_paths", help="a list of relative prefix path searchs of the form X=y:z,Y=p:q", default="", type=str
     )
     # only add compilers if this argument is passed
-    view_parser.add_argument("--compilers", help="path of the compilers.yaml file", type=str, default=None)
+    view_parser.add_argument("--compilers", help="path of the packages.yaml file", type=str, default=None)
```

Comment:
The `--compilers` name makes a bit less sense now that we look in `packages.yaml`, but I wasn't able to think of a better name yet. It is still used to find compilers, so it's not completely off... but suggestions for better names welcome.